FogCreek/Glitch-Community
# Remove sign-in-pop and redirect to /login

## Links
* Remix link: https://hissing-gibbon.glitch.me/
* Issue-tracker link: https://app.clubhouse.io/glitch/story/5180/redirect-to-glitch-com-login-from-community

## Changes:
* This removes the sign-in-pop component and uses a small button as an href to redirect to `/login` in order to handle sign-in. We're doing this to stay consistent with Editor and with the launch of a similar feature there.
* This also uses the sign-in-layout component to populate the OAuth sign-in page that is currently at `/signin`. I'm going to swap the `/login` and `/signin` routes in a second PR.

## How To Test:
* Be signed out. Click "Sign In" on the header (and recent projects) and see that you are taken to `/login` and not the popover.

## Feedback I'm looking for:

## Things left to do before deploying:
